Makefile

```diff
@@ -20,6 +20,8 @@ MAIL_SERVER ?= "smtp.gmail.com"
 MAIL_ADDRESS ?= "omeka@example.com"
 MAIL_PASSWORD ?= replace_me
 
+.PHONEY: init-app
```

Comment:
@nomadicoder I don't really understand what this is.  Can you explain it to me?

Reply:
@sensei100 We were getting a notification in app that there was a version update to the AltText plugin.  When I went to get the update information to change to version number in the plugin, there was a note that the AltText plugin was no longer needed, as the feature had been added to the latest version of Omeka-S, which we're already using.  Gabe verified this.

I also added an unzip option so it would overwrite files without waiting for a prompt for when we do future upgrades.

Reply:
@sensei100 and ensure we only clean up the zip files we downloaded.

Reply:
I understand now.  Github should open the expansion automatically so it's not buried.

Reply:
The `phony-init` was so the target gets triggered.  `make init-app` yielded `nothing to be done`.  However, it seems to be working without it.  I'll remove the definition.

Reference: https://unix.stackexchange.com/questions/403761/make-nothing-to-be-done-for-build:

Reply:
Resolved for sure